src/action/createDataSelectAction.js

```diff
@@ -50,7 +50,8 @@ export default function (seriesType, actionInfos) {
             );
             return {
                 name: payload.name,
-                selected: selected
+                selected: selected,
+                seriesId: payload.seriesId
```

Comment:
`payload` probably has no `seriesId`, and the action probably effect multiple series.

```js
dispatchAction({name: 'xxx'}); // Probably effect multiple series
dispatchAction({seriesName: 'xxx'}); // Probably effect multiple series
```

I think we'd better discuss more about the further enhancement of properties of events, which currently is not good enough. Some properties are missing.
And also we should consider that some common properties (like `seriesId` `seriesIndex` `componentIndex` `name`) should be consistent as far as possible.

Reply:
I have checked it again and I think this fix is correct. 
It can resolve the issue without negative effect to other cases.
The other cases can be discussed future.